harness/harness.go

```diff
@@ -104,7 +119,14 @@ func (c *H) flushToParent(format string, args ...interface{}) {
 
 	c.mu.Lock()
 	defer c.mu.Unlock()
-	io.Copy(p.w, &c.output)
+	outputBufferCopy := c.output
+	outputBufferCopy.WriteTo(p.w)
+}
+
+func (c *H) publishToReporters(name string, status reporters.TestResult) {
+	if !c.hasSub {
+		c.reporters.ReportTest(name, status, c.duration, c.output.Bytes())
```

Comment:
As a followup, we could potentially use something other than `c.output` here.

`c.output` came from an `indenter`. We don't need the indentation for reporters.

One way we could do that is by replacing `H.w` with a `TeeWriter` which tees to an indented sink and a normal sink, and then use whichever is appropriate.

We could also write verbosely to the 'reporter sink'.

I'm fine with this being a TODO if you don't want to tackle it in this initial PR.